services/ui_backend_service/data/cache/client/cache_server.py

```diff
@@ -234,26 +262,19 @@ def validate_actions(self, actions):
                                            % (mod_str, cls_str))
 
     def schedule(self):
-
         def queued_request(queue):
             while queue:
                 yield queue.popleft()
 
         for request in chain(queued_request(self.hi_prio_requests),
                              queued_request(self.lo_prio_requests)):
 
-            try:
```

Comment:
This PR will allow cache size to exceed `max_size` and rely on GC to cleanup old data once graceful period is over. 
I'm not sure which approach would be the better one.